draft-ietf-rtcweb-mdns-ice-candidates.md

```diff
@@ -251,6 +251,23 @@ same address and either the same or a different port; this includes cases
 where the signaled candidate is discarded as redundant according to 
 Section 5.1.3 of {{RFC8445}}.
 
+Interactions with TURN servers 
+------------------------------
+
+When sending data to a TURN {{RFC5766}} server, the sending client tells
+the server the destination IP and port for the data. This means that
+if the the client uses TURN to send to an IP that was obtained by mDNS
+resolution, the TURN server will learn the underlying host IP and port,
+and this information can then be relayed to the web application,
+defeating the value of the mDNS wrapping. 
+
+To prevent disclosure of the host IP address to a TURN server, the ICE
+agent MUST NOT form candidate pairs between its own relay candidates
+and remote mDNS candidates. Note that the converse is not an issue; the
+ICE agent MAY form a candidate pairs between its own mDNS candidates and
+remote relay candidates, as in this situation host IPs will not be sent
+directly to the TURN server.
```

Comment:
Interesting!
I am fuzzy of the practical impact of such a restriction.
It seems like it should not be limiting connectivity as the ICE agent can form candidate pairs between its relay candidates and remote self reflexive candidates.

Reply:
yeah, no practical impact.
Normally, you have:
- host->host, host->srflx, host->relay
- relay->host, relay->srflx, relay->relay

relay->host is what we're dropping, but I can't think of any scenario where that would work when the host candidate is private.

Reply:
I could mention that this has no practical impact.

Reply:
Paragraph added.